pkg/constants/env_variables.go

```diff
@@ -5,7 +5,7 @@ package constants
 
 // environment variables for http proxy
 const (
-	ProxyCACert = "PROXY_CA_CERT"
+	ProxyCACert = "TANZU_CLI_PROXY_CA_CERT"
```

Comment:
`TANZU_CLI_PROXY_CA_CERT` is used for self-signed CA and for the proxy use case?